src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/codeAction/proposal/quickfix/InsertAnnotationQuickFix.java

```diff
@@ -93,6 +96,8 @@ public CodeAction resolveCodeAction(JavaCodeActionResolveContext context) {
         try {
             WorkspaceEdit we = context.convertToWorkspaceEdit(proposal);
             toResolve.setEdit(we);
+        } catch (IndexNotReadyException | ProcessCanceledException | CancellationException e) {
```

Comment:
Noticed this is hitting Jakarta quick fixes too. We should make sure the new development on this branch: https://github.com/OpenLiberty/liberty-tools-intellij/tree/code-action-unification-2 is consistent is doing these re-throws where appropriate.